### ISSUE TITLE
Lack of type information leads to features configuration error

### ISSUE THREAD

**Issue by jashworth** [bryntum/support#2750]
Lack of full fidelity type information for GanttConfig causes the following TS error

![image](https://user-images.githubusercontent.com/4151375/115954446-00a3c480-a4e9-11eb-8c0e-3f6ec81740f8.png)

This can be addressed with

```typescript
const ganttConfig: Partial<GanttConfig & { [key: string]: unknown }> = {
  dependenciesFeature: false,
};
```

![image](https://user-images.githubusercontent.com/4151375/115954510-63955b80-a4e9-11eb-9974-28f1f86fa988.png)

but is no substitute for full fidelity types.

see also #2740

**Comment by SergeyMaltsev** [comment]
Features in react Wrapper are suffixed with `Feature` and act as both configs and properties for `BryntumGanttComponent`. 
They are mapped to the corresponding API features of the instance.
So those feature names from wrapper are not the part of API Types.

This is a valid GanttConfig
```
const ganttConfig: Partial<GanttConfig}> = {
   features : {
     dependencies: false
  }
};
```

Please check this issue for more info.
https://github.com/bryntum/support/issues/2751

**Comment by jashworth** [comment]
Your solution results in a deprecation warning.

See #2751

**Comment by SergeyMaltsev** [comment]
It is not deprecation but development warning which will not be show in production build.

**Comment by jashworth** [comment]
Either way, it's a warning about a configuration concern that, with better type information, could be delivered during development but, due to the lack of type information, isn't actually delivered until runtime, and that's the point of this issue.

**Comment by SergeyMaltsev** [comment]
Fixed released 4.1.2 version?
